cmake/Modules/CMakePolicy.cmake

```diff
@@ -10,3 +10,17 @@ endif()
 if(POLICY CMP0116)
   cmake_policy(SET CMP0116 OLD)
 endif()
+
++# Enable debug info compatible with SCCache
```

Comment:
At this point, this first line feels a bit out of place (and kinda redundant with the stuff on lines 20-23 below), can we just remove this one?

Reply:
Ping - if you'd have time to update this to remove the first comment line here, I'd be ok with merging this.

Reply:
Done